include/ocpp/v201/smart_charging.hpp

```diff
@@ -81,7 +81,12 @@ class SmartChargingHandler {
     ///
     /// \brief Adds a given \p profile and associated \p evse_id to our stored list of profiles
     ///
-    void add_profile(int32_t evse_id, ChargingProfile& profile);
+    ProfileValidationResultEnum add_profile(int32_t evse_id, ChargingProfile& profile);
+
+    ///
+    /// \brief Retrieves existing profiles on system.
+    ///
+    ChargingProfile get_profiles();
```

Comment:
This signature seems like it only returns a single profile instead of multiple profiles?